cluster/gce/ubuntu/node.yaml

```diff
@@ -0,0 +1,281 @@
+From nobody Tue Aug 11 10:13:54 2015
+Content-Type: multipart/mixed; boundary="===============6024533374511606659=="
+MIME-Version: 1.0
+
+--===============6024533374511606659==
```

Comment:
No, it is not for debugging. The configuration consists of multiple upstart jobs, and we need to pack them into one file and pass it to GCE metadata (the metadata key is "user-data" as shown in cluster/gce/ubuntu/helper.sh). Ubuntu cloud-init provides a tool https://github.com/number5/cloud-init/blob/master/tools/make-mime.py to pack upstart jobs into an MIME file. So, this header and boundary string are generated by that tool. Cloud-init then can parse the yaml file and extract each upstart job.